packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -323,6 +327,16 @@ class SearchAnchor extends StatefulWidget {
   /// To get a different layout, use [viewBuilder] to override.
   final SuggestionsBuilder suggestionsBuilder;
 
+  /// {@template flutter.widgets.SearchAnchor.textInputAction}
```

Comment:
Just pushed the change and approved the PR! Once all tests are passed, I will submit it! Thanks again for the improvement:)!

Reply:
> Just pushed the change and approved the PR! Once all tests are passed, I will submit it! Thanks again for the improvement:)!

Thank you! 🙌🙌

Reply:
> keyboardType also has one existing template

Yes, but I think using it directly will cause some failures because the doc uses `{maxlines}` which is a property of `TextField` but not a property of `SearchBar`